pages/linux/cmd.md

```diff
@@ -0,0 +1,15 @@
+# cmd
+
+> Android service manager.
+
+- List every running service:
+
+`cmd -l`
+
+- Call a specific service:
+
+`cmd {{alarm}}`
+
+- Call a service with arguments:
+
+`cmd {{vibrator}} {{vibrate 300}}`
```

Comment:
For clarity?
```suggestion
`cmd {{vibrator}} "{{vibrate 300}}"`
```

Reply:
This would not work, because these are separate arguments.

I guess the correct syntax would be this?
```suggestion
`cmd {{vibrator}} {{vibrate}} {{300}}`
```

Reply:
Ah, I see.  I read `Call a service with arguments` which tells me that the arguments and the amount of them are arbitrary—am I reading that right?

Reply:
The number of arguments depends on the service command called. Each service has its own set of commands that require different arguments.
Several examples:
```bash
cmd vibrator vibrate 300
cmd netpolicy set restrict-background true
cmd testharness enable
```

Reply:
Okay, makes sense.  I say leave it as is, then.  I think you did the right in the first place.

Reply:
Would two dashes work, e.. `cmd vibrator -- vibrate 300`?

Reply:
It does not seem to work:
```
Unknown command: --
```

Reply:
Okay, all settled.  Don't make any changes, then, and let's resolve this thread.